app/views/mentor/knowledges/new.html.haml

```diff
@@ -0,0 +1,7 @@
+= form_for ['mentor', @knowledge], remote: true do |f|
+  = f.text_field :title, placeholder: "title"
+  = f.text_area :content, cols: "100", row: "100"
+  = f.submit '送信'
+  = f.submit 'ajax', id: "ajax"
```

Comment:
__[HAML-Lint]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->